example_project_bzl_4/third_party/readline/no-arch_only.patch

```diff
@@ -0,0 +1,13 @@
+diff -ru -x '*~' readline-6.3-orig/support/shobj-conf readline-6.3/support/shobj-conf
```

Comment:
I believe that using `overrideAttrs` makes include of this file no longer nedeed